cmd/up/up.go

```diff
@@ -467,10 +471,16 @@ func (up *upContext) getCurrentDeployment(ctx context.Context, autoDeploy bool)
 		return nil, false, err
 	}
 
-	if !autoDeploy {
-		if err := utils.AskIfDeploy(up.Dev.Name, up.Dev.Namespace); err != nil {
-			return nil, false, err
+	if !up.Dev.Autocreate {
+		err = errors.UserError{
+			E: fmt.Errorf("Deployment %s not found in namespace '%s'", up.Dev.Name, up.Dev.Namespace),
+			Hint: `Please:
+	- Make sure your application is deployed and ready.
+	- Make sure your context is pointing to the right namespace.
+	- Set the 'autocreate' property in your okteto.yml manifest if you want to automatically create a new development container.
```

Comment:
```suggestion
- Set the 'autocreate' field in your okteto manifest if you want to automatically create a new deployment in your current namespace.
```

Reply:
What about a shorten version:
```suggestion
	- To automatically create a new deployment, set the 'autocreate' field in your okteto manifest.
```